arcgis-ios-sdk-samples/Geometry/Spatial relationships/SpatialRelationshipsViewController.swift

```diff
@@ -0,0 +1,242 @@
+        
+        // Add graphics overlay to mapView
+        mapView.graphicsOverlays.add(graphicsOverlay)
+        
+        // Set viewpoint to the point graphic geometry
+        if let point = pointGraphic.geometry as? AGSPoint {
+            mapView.setViewpointCenter(point, scale: 100000000.0, completion: nil)
+        }
+    }
+    
+    // MARK: GeoView Touch Delegate
+    
+    func geoView(_ geoView: AGSGeoView, didTapAtScreenPoint screenPoint: CGPoint, mapPoint: AGSPoint) {
+        //
+        // Identify graphics overlay
+        geoView.identify(graphicsOverlay, screenPoint: screenPoint, tolerance: 12, returnPopupsOnly: false, maximumResults: 1) { [weak self]  (result) in
+            //
+            // Make sure self is around
+            guard let strongSelf = self else {
+                return
+            }
+            
+            // Clear selection
+            strongSelf.graphicsOverlay.clearSelection()
+            
+            // Make sure there is no error
+            guard result.error == nil else {
+                SVProgressHUD.showError(withStatus: result.error?.localizedDescription)
+                return
+            }
+            
+            // Return if there is no graphic identified or geometry is not available
+            guard let identifiedGraphic = result.graphics.first, let selectedGeometry = identifiedGraphic.geometry else {
+                return
+            }
+            
+            // Select identified graphic
+            identifiedGraphic.isSelected = true
+            
+            // Setup result view controller
+            let storyboard = UIStoryboard(name: "ExpandableTableViewController", bundle: nil)
+            let expandableTableViewController = storyboard.instantiateViewController(withIdentifier: "ExpandableTableViewController") as! ExpandableTableViewController
+            
+            // Create objects to store results
+            var pointRelationships = [String]()
+            var polylineRelationships = [String]()
+            var polygonRelationships = [String]()
+            
+            // Check the geometry type and find it's
+            // relationship with other geometries
+            if let polylineGeometry = strongSelf.polylineGraphic.geometry, let polygonGeometry = strongSelf.polygonGraphic.geometry, selectedGeometry.geometryType == .point {
+                //
+                // Get the relationships with polyline and polygon
+                polylineRelationships = strongSelf.getSpatialRelationships(a: selectedGeometry, b: polylineGeometry)
+                polygonRelationships = strongSelf.getSpatialRelationships(a: selectedGeometry, b: polygonGeometry)
+                
+                // Add section for polyline relationships and it's items
+                expandableTableViewController.sectionHeaderTitles.append("Relationship With Polyline")
+                let polylineSectionItems = polylineRelationships.map { ($0, "") }
```

Comment:
We need `[(String, String)]` for the `expandableTableViewController`. I feel, it will be confusing for `getSpatialRelationships` function to return `[(String, String)]`. 

The heart of the sample is `getSpatialRelationships` so renaming function to `makeSectionItems` and returning `[(String, String)]` will take away the message/meaning of the sample. 

Let's keep this code as is.